.gitignore

```diff
@@ -41,3 +42,6 @@ docs/changes/
 docs/latex/
 libdrizzle-config
 test-suite.log
+
+# Keep empty directories:
+# Keep empty directories: >> .gitignore/.git*
```

Comment:
I would find out for sure or just remove it, but adding stuff that we don't know what is it for doesn't seem like a good idea :)